lib/price_calculator.rb

```diff
@@ -0,0 +1,37 @@
+require "terminal-table"
+
+require_relative './cart'
+require_relative './invoice'
+
+class PriceCalculator 
+
+    attr_reader :items
+
+    def initialize
+        @cart = Cart.new.items_bought_list
```

Comment:
cart variable should only contain instance of the Cart class. Any action related to it must be done using the instance of that class.